src/win32/w32_util.c

```diff
@@ -150,8 +152,18 @@ size_t git_win32__canonicalize_path(wchar_t *str, size_t len)
 	/* "\??\UNC\", "\\?\UNC\" -- UNC prefix */
 	if (to_advance && len >= CONST_STRLEN(unc_prefix) &&
 		!wcsncmp(str + to_advance, unc_prefix, CONST_STRLEN(unc_prefix))) {
-		to_advance += CONST_STRLEN(unc_prefix);
-		len -= CONST_STRLEN(unc_prefix);
+		/** 
```

Comment:
`/** */` style comments are only used for strings that should show up in our documentation. But these comments here are only really relevant to developers of libgit2, not users of our API. So these should instead be `/* */` comments.

Reply:
Understood.